src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/TaskControllerTest.java

```diff
@@ -1,112 +1,147 @@
 package uk.gov.hmcts.reform.wataskmanagementapi.controllers;
 
-import net.serenitybdd.junit.spring.integration.SpringIntegrationSerenityRunner;
-import org.eclipse.jetty.http.HttpStatus;
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import org.junit.Before;
 import org.junit.Test;
-import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatus;
 import org.springframework.http.MediaType;
+import uk.gov.hmcts.reform.wataskmanagementapi.SpringBootFunctionalBaseTest;
+import uk.gov.hmcts.reform.wataskmanagementapi.domain.entities.camunda.CamundaTask;
+import uk.gov.hmcts.reform.wataskmanagementapi.services.CcdIdGenerator;
 
+import java.util.List;
 import java.util.UUID;
 
+import static net.serenitybdd.rest.SerenityRest.expect;
 import static net.serenitybdd.rest.SerenityRest.given;
+import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.notNullValue;
 
-@RunWith(SpringIntegrationSerenityRunner.class)
-public class TaskControllerTest {
+public class TaskControllerTest extends SpringBootFunctionalBaseTest {
```

Comment:
This is ok for now, but may be better in future to have each functional test deal with one API endpoint.  That way it can grow to cover all the error scenarios etc.